docs/examples/oplsaa_with_silica.xml

```diff
@@ -0,0 +1,2356 @@
+<ForceField>
```

Comment:
Could we trim this file? The other XML file looks clean and easy to read, this one does not. #161 should be able to do most of the work

Reply:
I tried that, actually. However, [the lj scaling inference kept returning exceptions](https://github.com/mosdef-hub/foyer/pull/161/files#diff-bea6c053128dd6e37cbf4cef50a18b4eR250). Ill go through it with some regex and grab the only relevant parameters, unless you might know a better way to skip this error?

Reply:
That sounds like a case not currently handled by the logic of the function as-written. I can look at this later today

Reply:
This might be something to deal with another day